src/components/NoteWhiteArea.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { note } from './NoteWhiteArea.scss';
+
+const NoteWhiteArea = ({ className, children, ...props }) =>
```

Comment:
Some of forms (ex [here](https://projects.invisionapp.com/d/main#/console/11184881/237600095/preview)) contain areas with description or comments. So I thought it makes sense to make separate component from it. Technically it could be just separate class in `common.scss` but I thought logically its more a component than just style declaration.